apps/store/src/components/CartPage/CartPage.tsx

```diff
@@ -76,7 +76,7 @@ const EmptyState = ({ children, prevURL }: EmptyStateProps) => {
         <Space y={2}>
           <Space y={1}>
             <CenteredText>¯\_(ツ)_/¯</CenteredText>
-            <CenteredText>{t('CART_EMPTY_SUMMARY')}</CenteredText>
+            <CenteredTextSecondary>{t('CART_EMPTY_SUMMARY')}</CenteredTextSecondary>
```

Comment:
I think we can just use props for this nowadays :D

```suggestion
            <Text align="center" color="textSecondary">{t('CART_EMPTY_SUMMARY')}</Text>
```